components/content/inspira/ui/aurora-background/AuroraBackground.vue

```diff
@@ -0,0 +1,58 @@
+<script setup lang="ts">
+import { cn } from "@/lib/utils";
+
+interface AuroraBackgroundProps {
+  radialGradient?: boolean;
+  class?: string;
+}
+
+const props = withDefaults(defineProps<AuroraBackgroundProps>(), {
+  radialGradient: true,
+});
+</script>
+
+<template>
+  <main>
+    <div
+      v-bind="props"
+      :class="
+        cn(
+          'relative flex flex-col h-[100vh] items-center justify-center bg-zinc-50 dark:bg-zinc-900 text-slate-950 transition-bg',
+          props.class,
+        )
+      "
+    >
+      <div class="absolute inset-0 overflow-hidden">
+        <div
+          class="aurora-background-gradient"
+          :class="
+            cn(
+              props.radialGradient &&
+                `[mask-image:radial-gradient(ellipse_at_100%_0%,black_10%,var(--transparent)_70%)]`,
+            )
+          "
+        ></div>
+      </div>
+      <slot />
+    </div>
+  </main>
+</template>
+
+<style scoped>
+.aurora-background-gradient {
```

Comment:
Can we make it more readable?
Something like `cn('<css-style>', '<before:css-style>`, '<after:css-style>', '<animation-css-style>')`.
This makes it more readable, implemented in Magic UI also.

Reply:
This make more scence let me check, Thanks @rahul-vashishtha

Reply:
@rahul-vashishtha, did the changes as per my understanding, please check once and let me know if nay changes required.